frontend/XCITE/Nodes/Layout.qml

```diff
@@ -27,88 +29,127 @@ ColumnLayout {
     }
     RegistrationLevel {
         visible: tracker == 1
+        Label {
+            id: back1
 
-        Controls.ButtonModal {
-            Layout.fillHeight: true
+            readonly property string defaultText: qsTr("< Back")
+            property bool isActive: false
+            anchors.top: parent.top
             anchors.left: parent.left
-            anchors.bottom: parent.bottom
-            isPrimary: true
-            width: 300
-            buttonHeight: 50
-            label.font.weight: Font.Medium
-            label.font.letterSpacing: 3
-            label.text: qsTr("<- Back")
-
-            onButtonClicked: {
-                tracker = 0
+            topPadding: 60
+            leftPadding: 35
+            font.pixelSize: 12
+            font.weight: isActive ? Font.Bold : Font.Normal
+            text: isActive ? altText : defaultText
+            color: "#0ED8D2"
+
+            MouseArea {
+                anchors.fill: parent
+                cursorShape: Qt.PointingHandCursor
+                onClicked: {
+                    tracker = 0
+                }
             }
         }
+        Label {
+            id: regdetails
+
+            readonly property string defaultText: qsTr("Registration Details >")
+            property bool isActive: false
 
-        Controls.ButtonModal {
-            Layout.fillHeight: true
             anchors.right: parent.right
-            anchors.bottom: parent.bottom
-            isPrimary: true
-            width: 350
-            buttonHeight: 50
-            label.font.weight: Font.Medium
-            label.font.letterSpacing: 3
-            label.text: qsTr("   Registration Details ->")
-
-            onButtonClicked: {
-                tracker = 2
+            font.pixelSize: 12
+            font.weight: isActive ? Font.Bold : Font.Normal
+            text: isActive ? altText : defaultText
+            color: "#0ED8D2"
```

Comment:
Could you use Theme for these color constants where they've been defined? 
`XCITE/Home/ChartDiode.qml` would be a good example of importing/using the `Theme`